### DISCUSSION TITLE
Email notifications when a draft pull request is marked as Ready for review

### DISCUSSION THREAD

**Discussion by mlocati** [community/community#22826] [New to GitHub]
<p>Hi all,</p>
<p>does GitHub sends notifications to the project watchers when a pull request created as Draft is marked as Ready For Review?</p>
<p>I’m asking because this seems to me an important feature, but it doesn’t seem that GitHub does that…</p>
<p>–<br>
Michele</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hey there <a class="mention" href="/mlocati">@mlocati</a> :waffle:</p>
<p>If you are listed as an assigned reviewer directly or via a team, you will get a notification to review a pull request.</p>
<p>We do not have a general notification for watchers that cannot actually complete a review.<br>
Here’s more info on <a href="https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-code-review-assignment-for-your-team">Code Review assignments</a>.</p>
<p>We’re always working to improve GitHub and the GitHub Support Community, and we consider every suggestion we receive.<br>
Would you mind <a href="https://support.github.com/contact/feedback">submitting this through our official product feedback form</a> so that our product team can track your request?</p>

**Comment by mlocati** [comment]
<p>Thank you <a class="mention" href="/ernest-phillips">@ernest-phillips</a>! I’ve just submitted a feedback!</p>

**Comment by native-api** [comment]
I second this feature request. As a team member, I am able to review any pull request, whether I'm assigned or not. Our project does not have tons of pull requests so it's not necessary to explicitly assign people to them to avoid duplication of effort. Moreover, even people not on the team can review -- their review will just be marked with a pale tick rather than a bright green tick and will not count if the repository is configured to require reviews before merging.

Not receiving notifications has repeatedly caused delays measured in days before a pull request that was submitted as draft is merged, for absolutely no good reason -- just because I have better things to do than checking on things manually every day.

So I see absolutely no reason to not send notifications. At the very least, this should be a configurable option among the other notification types.

**Reply by native-api** [reply]
[Submitted feedback](https://github.com/orgs/community/discussions/158714)
